cli_test/cli_posts_test.go

```diff
@@ -38,7 +38,7 @@ func TestDesmosCLIPostsCreateNoMediasNoPollData(t *testing.T) {
 	tests.WaitForNextNBlocksTM(1, f.Port)
 
 	// Make sure the post is saved
-	storedPosts := f.QueryPosts()
+	storedPosts := f.QueryPosts("--hashtag=#hashtag")
```

Comment:
I think the `#` here should be deleted in order to be similar to what happens with the REST APIs:

```suggestion
	storedPosts := f.QueryPosts("--hashtag=hashtag")
```